{{cookiecutter.repository_name}}/.github/workflows/tests.yml

```diff
@@ -0,0 +1,67 @@
+name: Run Tests
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: {{ '${{ matrix.os }}' }}
+    strategy:
+      max-parallel: 4
+      matrix:
+        python-version: [3.5, 3.6, 3.7, 3.8]
+        os: [ubuntu-latest, macos-latest]
+
+    steps:
+    - uses: actions/checkout@v1
+    - name: Set up Python {{ '${{ matrix.python-version }}' }}
+      uses: actions/setup-python@v1
+      with:
+        python-version: {{ '${{ matrix.python-version }}' }}
+        
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install tox tox-gh-actions
+
+    - name: Test with tox
+      run: tox
+
```

Comment:
I would put the blank lines inside the `if` block, so they are only added when the conditions evaluate to true.

Reply:
The whitespace before the `{%-` is removed at compile time since we specified `-`, I think the current behavior is correct?

Reply:
Oh, my bad. If that's the case yes, it's already correct.